packages/labs/react/README.md

```diff
@@ -55,6 +55,49 @@ React component.
 />
 ```
 
+#### Typescript
+
+Event callbacks can be typed using the included `EventHandler` type. By using a
+typecast, `createComponent` correlates callback types to property names from an event property map.
+
+Uncasted EventHandlers will fallback to `(e: Event) => void`.
```

Comment:
The fallback is that the event type is `Event`, right?

```suggestion
Non-casted event names will fallback to an event type of `Event`.
```

Reply:
Done :)